integration/unplugin/src/index.ts

```diff
@@ -215,15 +223,22 @@ const civetUnplugin = createUnplugin((options: PluginOptions = {}) => {
     },
     resolveId(id, importer) {
       if (/\0/.test(id)) return null;
-      if (!isCivet(id) && !isCivetTranspiled(id)) return null;
 
       id = cleanCivetId(id);
       const absolutePath = path.isAbsolute(id)
         ? resolveAbsolutePath(rootDir, id)
```

Comment:
I think it would fail here for absolute imports with implicit civet extension. I'm not sure if we want to account for that case though?

Reply:
Nice catch! I've pushed a patch, which I tested and seems to work for both types of absolute paths (at least in Vite). Could you review?

Reply:
Yep this looks good!